package.json

```
@@ -61,8 +61,8 @@
     "release:changelog": "node scripts/releaseChangelog.mjs",
     "release:version": "lerna version --exact --no-changelog --no-push --no-git-tag-version",
     "release:build": "lerna run --parallel --scope \"@mui/*\" build",
-    "release:publish": "lerna publish from-package --dist-tag next --contents build",
-    "release:publish:dry-run": "lerna publish from-package --dist-tag next --contents build --registry=\"http://localhost:4873/\"",
+    "release:publish": "lerna publish from-package --dist-tag master --contents build",
+    "release:publish:dry-run": "lerna publish from-package --dist-tag master --contents build --registry=\"http://localhost:4873/\"",
```

NPM packages are released as either `next` or `latest`. 🙈 
```suggestion
    "release:publish": "lerna publish from-package --dist-tag latest --contents build",
    "release:publish:dry-run": "lerna publish from-package --dist-tag latest --contents build --registry=\"http://localhost:4873/\"",
```